addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6488,6 +6505,7 @@ registry.ImageTools = ImageHandlerOption.extend({
             delete img.dataset.fileName;
             delete img.dataset.shapeFlip;
             delete img.dataset.shapeRotate;
+            delete img.dataset.animationRatio;
```

Comment:
This should also be removed when switching from an animated shape to an non-animated one (see how we did for `img.dataset.hoverEffect` ...